source/query/project.txt

```diff
@@ -0,0 +1,90 @@
+.. _query-bar-project:
+
+=============================
+Set Which Fields Are Returned
+=============================
+
+.. default-domain:: mongodb
+
+If the query bar displays the :guilabel:`Project` option, you can
+specify which fields to return in the resulting data. By default, all
+fields are returned.
+
+To set a projection:
+
+#. Click :guilabel:`Options`.
+
+#. Enter the projection document into the :guilabel:`Project` field.
+
+   To include fields:
+     Specify the field name and set to ``1`` in the project document.
+
+     .. example::
+
+        .. code-block:: javascript
+
+           { year: 1, name: 1 }
+
+        Only the fields specified in the project document are returned.
+        The ``_id`` field is returned unless it is set to  ``0`` in
+        project document.
+
+   To exclude fields:
+     Specify the field name and set to ``0`` in the project document.
+
+     .. example::
+
+        .. code-block:: javascript
+
+           { year: 0, name: 0 }
+
+        All fields except for the fields specified in the project
+        document are returned.
+
+   As you type, the :guilabel:`Find` button is disabled and the
+   :guilabel:`Project` label turns red until a valid query is entered.
+
+   .. figure:: /images/querybar/query-project-error.png
+      :figwidth: 300px
+      :lightbox:
+
+#.
+   .. include:: /includes/run-query.rst
+
+   .. figure:: /images/querybar/query-project-success.png
+      :figwidth: 300px
+      :lightbox:
+
+To Learn More
+-------------
+
+To learn how project works, see the ``project`` entry in the
+:ref:`MongoDB Manual <find-projection>`.
+
+How Do MongoDB and Compass Compare to SQL?
```

Comment:
Should we be a bit more specific on this title? It sounds like we are comparing the product(s) (MDB, Compass) to a language (SQL)?